slides/04-evaluating-models.qmd

```diff
@@ -485,46 +648,36 @@ countdown(minutes = 8, id = "try-fit-resamples")
 ## Evaluating model performance `r hexes("tune")`
 
 ```{r}
-ctrl_frog <- control_resamples(save_pred = TRUE)
+ctrl_taxi <- control_resamples(save_pred = TRUE)
 
 # Random forest uses random numbers so set the seed first
 
 set.seed(2)
-rf_res <- fit_resamples(rf_wflow, frog_folds, control = ctrl_frog)
+rf_res <- fit_resamples(rf_wflow, taxi_folds, control = ctrl_taxi)
 collect_metrics(rf_res)
 ```
 
-## 
-
-```{r}
-collect_predictions(rf_res) %>% 
-  ggplot(aes(latency, .pred, color = id)) + 
-  geom_abline(lty = 2, col = "gray", size = 1.5) +
-  geom_point(alpha = 0.5) +
-  coord_obs_pred()
-```
-
 ## How can we compare multiple model workflows at once? {background-image="images/Hatching-process.jpg"}
```

Comment:
Can we have two different taxi svgs? Even one with spinning wheels and one static would be okay :) (Unless we remove that section, of course.)

Reply:
There's currently `taxi.png` and `taxi_spinning.svg` on `nyr-2023`. :)